go.mod

```diff
@@ -33,4 +34,5 @@ replace (
 	bitbucket.org/ww/goautoneg => github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
```

Comment:
this line (and the comment) can be removed now, given that library-go is getting updated in this PR

Reply:
@pedjak comment addressed